front/lib/tracking/node.ts

```diff
@@ -0,0 +1,117 @@
+import type {
```

Comment:
Not a massive fan of `node` as a name but that's definitely a total nit. What's the rational for not using `server` ?

Reply:
I just re-used what was already done for amplitude. I also prefer `server` and `client`. Updating (both for amplitude and the tracking lib) 👍

Reply:
done